### PR TITLE
feat: Support Multi-Cluster workflow execution - POC

### PR DESCRIPTION
**Pull request by sarabala1979**
Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.
* [ ] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [ ] I've signed the CLA.
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My builds are green. Try syncing with master if they are not. 
* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo/blob/master/USERS.md).

### CODE REVIEW

File: cmd/workflow-controller/main.go [hunk 1]
```diff
@@ -79,12 +82,39 @@ func NewRootCommand() *cobra.Command {
...
 			// start a controller on instances of our custom resource
-			wfController, err := controller.NewWorkflowController(config, kubeclientset, wfclientset, namespace, managedNamespace, executorImage, executorImagePullPolicy, containerRuntimeExecutor, configMap)
+			wfController, err := controller.NewWorkflowController(config, kubeclientset, wfClientset, namespace, managedNamespace, executorImage, executorImagePullPolicy, containerRuntimeExecutor, configMap)
 			errors.CheckError(err)
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 
 			go wfController.Run(ctx, workflowWorkers, podWorkers)
+			fmt.Println("Reading map")
+			kubeCM, err := kubeclientset.CoreV1().ConfigMaps(namespace).Get("clusterconfig", v1.GetOptions{})
```
Comment (sarabala1979) [inline cmd/workflow-controller/main.go:92]: Configure other clusters kubeconfig in Configmap

File: cmd/workflow-controller/main.go [hunk 2]
```diff
@@ -79,12 +82,39 @@ func NewRootCommand() *cobra.Command {
...
+				if err != nil {
+
+				}
+				fmt.Println(filename)
+				config1, err := clientcmd.BuildConfigFromFlags("", filename)
+				kubeclientset1 := kubernetes.NewForConfigOrDie(config1)
+				wfclientset1 := wfclientset.NewForConfigOrDie(config1)
+				wfController1, err := controller.NewWorkflowController(config1, kubeclientset1, wfclientset1, "argo", managedNamespace, executorImage, executorImagePullPolicy, containerRuntimeExecutor, configMap)
+				errors.CheckError(err)
+				go func() {
```
Comment (sarabala1979) [inline cmd/workflow-controller/main.go:112]: Start to Goroutine for each cluster

**Comment by alexec** [comment]
Can we hold off from this? I think we should consider going bigger.

**Comment by alexec** [comment]
No activity in 4 weeks. Shall we close this for the moment?

**Pull request closed without merging**